commitlint.config.js

```diff
@@ -20,9 +20,10 @@ module.exports = {
 				"stark-core",
 				"stark-demo",
```

Comment:
This one should be removed too right?

Reply:
Yes, you're right. 
Btw, shouldn't we also remove the "build" or the "build-main" ?
I have the feeling those 2 scopes are for the same thing 😕 

What do you think @nicanac @christophercr ?

Reply:
Yep, I think they are referring to the same thing... so maybe we can just keep "build-main"? Because that one is more specific and can't be easily overlooked than "build" which could be confused with "stark-build". Moreover, we already have a "build-main" label in the Github repo ;)